apps/ejabberd/src/ejabberd_auth_jwt.erl

```diff
@@ -200,11 +214,12 @@ remove_user(_LUser, _LServer, _Password) ->
     {error, not_allowed}.
 
 
-%% Internal helper
+%% Internal helpers
+
 get_auth_opt(Host, Key) ->
```

Comment:
`get_auth_opt` and `set_auth_opts` are generic for all backends, so possible code duplication.
Maybe we should move in into `ejabberd_auth`.